src/main/drivers/motor.c

```diff
@@ -206,7 +217,16 @@ void motorDevInit(const motorDevConfig_t *motorConfig, uint16_t idlePulse, uint8
     case PWM_TYPE_DSHOT600:
     case PWM_TYPE_DSHOT1200:
     case PWM_TYPE_PROSHOT1000:
-        motorDevice = dshotPwmDevInit(motorConfig, idlePulse, motorCount, useUnsyncedPwm);
+#ifdef USE_DSHOT_BITBANG
+        if (motorConfig->useDshotBitbang == DSHOT_BITBANG_ON ||
+            (motorConfig->useDshotBitbang == DSHOT_BITBANG_AUTO && motorConfig->useDshotTelemetry)) {
+            motorDevice = bbshotDevInit(motorConfig, idlePulse, motorCount, useUnsyncedPwm);
```

Comment:
`dshotBitbangDevInit()` matches the naming below.

Reply:
done